bin/connect-distributed.sh

```diff
@@ -23,13 +23,16 @@ fi
 base_dir=$(dirname $0)
 
 if [ "x$KAFKA_LOG4J_OPTS" = "x" ]; then
+    echo "DEPRECATED: using log4j 1.x configuration. To use log4j 2.x configuration, run with: 'export KAFKA_LOG4J_OPTS=\"-Dlog4j.configurationFile=file:$base_dir/../config/connect-log4j2.properties\"'"
     export KAFKA_LOG4J_OPTS="-Dlog4j.configuration=file:$base_dir/../config/connect-log4j.properties"
```

Comment:
Why don't you change this (and all the similar cases) to the log4j2 property? If I get it right then after the upgrade `-Dlog4j.configuration` won't work anyway so while I see the value of notifying the user, I think we should just use the log4j2 property outright. Or is there a backward compatibity, so does `log4j.configuration` work under log4j2?

Reply:
Yes, if you set log4j.configuration to reference a log4j.xml (Log4j 1 configuration) and you have log4j-1.2-api on the classpath then Log4j 2 will process the log4j 1 configuration. We improve this support with every release. Substantial improvements will be in 2.17.2 which should come out by the end of the month.

Reply:
@rgoers Thanks for the clarification. It seems like we also have to upgrade to log4j 2.17.2 soon.